Makefile.inc1

```diff
@@ -2054,8 +2047,8 @@ PKGMAKEARGS+=	PKG_VERSION=${PKG_VERSION} \
 packages: .PHONY
 	${_+_}${MAKE} -C ${.CURDIR} ${PKGMAKEARGS} real-packages
 
-update-packages: .PHONY
-	${_+_}${MAKE} -C ${.CURDIR} ${PKGMAKEARGS} real-update-packages
+update-packages: stage-packages .PHONY
```

Comment:
The `update-packages` target continues to have the same behaviour as before. The only difference is that the `real-update-packages` target does not force a stage and creation of world, kernel, AND source package lists.

NOTE: `stage-packages` is listed in the prerequisites because it does not depend on the value of the `PKG_VERSION` variable, whereas `create-packages` does.